oss_src/unity/lib/unity_sarray.cpp

```diff
@@ -1622,6 +1622,16 @@ std::shared_ptr<unity_sarray_base> unity_sarray::sample(float percent,
   return logical_filter(seq);
 }
 
+
+std::shared_ptr<unity_sarray_base> unity_sarray::hash(int random_seed) {
```

Comment:
Ah, okay.  Let's maybe then take it and do `seed = hash(seed) % 2**31` at the python level.